packages/maskbook/src/components/InjectedComponents/PostDialogHint.tsx

```diff
@@ -14,16 +16,38 @@ export interface PostDialogHintUIProps {
 
 const useStyles = makeStyles((theme) => ({
     button: {
-        padding: '8px',
+        padding: isMobileFacebook ? 0 : '8px',
+    },
+    text: {
+        fontSize: 14,
+        color: '#606770',
+        marginLeft: theme.spacing(1),
+    },
+    wrapper: {
+        display: 'flex',
+        alignItems: 'center',
+        width: '100%',
+        padding: '8px 10px',
+        borderBottom: '1px solid #dadde1',
     },
 }))
 
 export const PostDialogHintUI = memo(function PostDialogHintUI(props: PostDialogHintUIProps) {
     const classes = useStyles()
-    return (
+    const { t } = useI18N()
+    const EntryIconButton = memo((props: PostDialogHintUIProps) => (
```

Comment:
IMO this usage of memo is wrong. please remove it

Reply:
And this creates a new different React component everytime

Reply:
done